docs/database/overview.mdx

```diff
@@ -27,23 +27,23 @@ values={[
 
 :::caution
 This package version is not null safe migrated yet, but has been implemented to be compatible with other null safe FlutterFire packages.
-With that in mind, you will need to use the `beta` channel which supports null safe projects.
+Null safety is now available in Flutter's `stable` channel for Web, Android & iOS targets.
```

Comment:
```suggestion
```

Don't think we need this now, we say roughly the same thing again a few lines down.